src/DynamoCoreWpf/Views/LibrarySearchView.xaml.cs

```diff
@@ -537,28 +611,68 @@ private ListBoxItem GetListItemByIndex(ListBox parent, int index)
             return null;
         }
 
-
         // Everytime, when top result is updated, we have to select one first item.
         private void OnTopResultTargetUpdated(object sender, DataTransferEventArgs e)
         {
             // If we turn to regular view, we have to hide tooltip immediately.
             if ((DataContext as SearchViewModel).CurrentMode !=
-                Dynamo.ViewModels.SearchViewModel.ViewMode.LibrarySearchView)
+                SearchViewModel.ViewMode.LibrarySearchView)
             {
-                libraryToolTipPopup.DataContext = null;
+                libraryToolTipPopup.SetDataContext(null, true);
```

Comment:
I thought we got rid of `DataContext = null` a long time ago? Maybe this was due to a merge issue and got lost.